setup.cfg

```diff
@@ -4,7 +4,5 @@ dev = develop easy_install lasagne[testing]
 [pytest]
 addopts =
     -v --doctest-modules
-    --cov=lasagne --cov-report=term-missing
```

Comment:
It's good with respect to Travis, but it also means that users cannot simply enter `py.test`, but have to run the full command as in `.travis.yml`, though. I'm not sure we want that. It makes it more intuitive to just test a single (you don't need to know about `-k`), but harder to run the full test suite. Isn't this more of a bug in PyCharm?